docs/engineering/start-developing/bobaeditor.md

```diff
@@ -2,15 +2,17 @@
 sidebar_position: 3
 ---
 
-# Install BobaEditor
+# Getting Started: BobaEditor
 
 \[[code](https://github.com/essential-randomness/boba-editor), [demo](https://bobaeditor.netlify.app/?path=/story/editor-preview--simple-editor)]
 
 BobaEditor is BobaBoard's own extension of the [QuillJS text editor](https://quilljs.com/), and is responsible for anything related to **text formatting** and **embeds** (both in "edit" and "display" mode).
 
-## Install Instructions
+## Check Out the Codebase 
 
-In a terminal, run the following commands:
+First, [fork the boba-editor repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) in order to save a copy of the code to your own account.
```

Comment:
If you add these, then the clone command won't be `https://github.com/essential-randomness/boba-editor.git` but `https://github.com/YOUR_GITHUB_ACCOUNT/boba-editor.git`

Reply:
oh right yes